src/components/MayHaveMissed.vue

```diff
@@ -18,8 +18,11 @@
   // Use async/await to handle asynchronous behavior
   async function fetchArticleData() {
     try {
-      const result = await chocolateSauce(hashtagBuildTheList(articleNumber));
-      articleArray.value = result;
+      const listItem: string | Array<string> | undefined = hashtagBuildTheList(articleNumber);
+      if (typeof listItem === "string") {
+        const result = await chocolateSauce(listItem);
+        articleArray.value = result;
+      }
     } catch (err) {
       console.log(err);
     }
```

Comment:
Would be nice moving forward to also output a fun front-facing something like "WHOOPS! WHOOPSIES!" for a few reasons. would be nice to not have to dig through inspector, while also adding WHIMSY